starboard/build/config/BUILDCONFIG.gn

```diff
@@ -216,6 +217,14 @@ template("target_with_platform_configs") {
       configs += [ "//starboard/build/config:pedantic_warnings" ]
     }
     configs += [ toolchain_config_path ]
+
+    # if (sb_is_modular && current_toolchain == cobalt_toolchain) {
```

Comment:
this is weird, i dont see the change locally when i checkout this branch

Reply:
Let's just make sure this doesn't end up in trunk

Reply:
will do! still dont see it when i fetch the upstream change